sale_invoice_delivery_state/views/partner.xml

```diff
@@ -0,0 +1,11 @@
+<odoo>
+    <record id="view_partner_property_form" model="ir.ui.view">
+        <field name="model">res.partner</field>
+        <field name="inherit_id" ref="account.view_partner_property_form" />
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='property_payment_term_id']" position="after">
+                <field name="invoice_policy" />
+            </xpath>
```

Comment:
To be consistent with other modules like account_invoice_base_invoicing_mode, I'll advise to put the new field in the accounting page.